lib/jekyll/tags/include.rb

```diff
@@ -55,31 +55,28 @@ def parse_params(context)
 
       def validate_file_name(file)
         if file =~ INVALID_SEQUENCES || file !~ VALID_FILENAME_CHARS
-          raise ArgumentError, <<-MSG
-Invalid syntax for include tag. File contains invalid characters or sequences:
+          raise ArgumentError, <<~MSG
+            Invalid syntax for include tag. File contains invalid characters or sequences:
+             #{file}
+             Valid syntax:
+             #{syntax_example}
 
-  #{file}
-
-Valid syntax:
-
-  #{syntax_example}
-
-MSG
+          MSG
```

Comment:
This change is breaking tests and causing Travis to fail.

Reply:
Thanks, fixed.